From 6ea353b34d39b92ef3105e08a031848b17236e76 Mon Sep 17 00:00:00 2001
From: Victoria Milhoan <vicki.milhoan@freescale.com>
Date: Tue, 27 Jan 2015 00:16:23 -0700
Subject: [PATCH] MLK-10159 Use macros for CAAM clock numbers in imx6sx DTS
 file

Use macros instead of numbers to represent CAAM clocks in the
imx6sx device tree source.

Signed-off-by: Victoria Milhoan <vicki.milhoan@freescale.com>
---
 arch/arm/boot/dts/imx6sx.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 39502ce..8c83fb1 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -885,7 +885,8 @@
 				reg = <0x2100000 0x40000>;
 				ranges = <0 0x2100000 0x40000>;
 				interrupt-parent = <&intc>; /* interrupts = <0 92 0x4>; */
-				clocks = <&clks 134>, <&clks 135>, <&clks 136> ,<&clks 213>;
+				clocks = <&clks IMX6SX_CLK_CAAM_MEM>, <&clks IMX6SX_CLK_CAAM_ACLK>,
+					 <&clks IMX6SX_CLK_CAAM_IPG> ,<&clks IMX6SX_CLK_EIM_SLOW>;
 				clock-names = "caam_mem", "caam_aclk", "caam_ipg", "caam_emi_slow";
 
 				sec_jr0: jr0@1000 {
-- 
1.8.0

